src/Connections/GenericHook.ts

```diff
@@ -620,6 +621,10 @@ export class GenericHookConnection
         msg.html = `<strong>${safeData.username}</strong>: ${msg.html}`;
       }
     }
+
+    if (typeof safeData?.relates_to  === "object") {
+      msg.relates_to = safeData.relates_to ?? undefined;
```

Comment:
I think this needs stronger validation, at least checking whether the object looks like a relates_to object?